scripts/ci/roxctl.sh

```diff
@@ -27,7 +27,7 @@ main() {
     fi
     api_endpoint="${api_hostname}:${api_port}"
 
-    roxctl -e "${api_endpoint}" --insecure-skip-tls-verify "$@"
+    roxctl -s "" -e "${api_endpoint}" --insecure-skip-tls-verify "$@"
```

Comment:
The `-s ""` looks really weird. Is it used because of a default would be provided when we omit `-s`?

Reply:
Okay, I saw this explanation:

> This change is driven by the CI failures where roxctl is confused with the server name set in the environment. So I added -s "" to ignore the environment and connect to the endpoint server name.

but that feels like using this script is doomed to ignore the `ROX_SERVER_NAME` without raising any warnings to the caller. As that maybe considered as a silent failure, I would suggest to make it a loud failure by adding something like:

```
    if [ -n "${ROX_SERVER_NAME}" ]; then
        echo "Provided value of ROX_SERVER_NAME will be ignored within this script."
    fi
```

in the top section.